mkosi/__init__.py

```diff
@@ -829,24 +829,22 @@ def reuse_cache_image(
 
 
 @contextlib.contextmanager
-def attach_image_loopback(
-    args: CommandLineArguments, raw: Optional[BinaryIO]
-) -> Generator[Optional[Path], None, None]:
-    if raw is None:
+def attach_image_loopback(image: Optional[BinaryIO]) -> Generator[Optional[Path], None, None]:
```

Comment:
btw, just wondering, does python support giving names to return type annotations? i mean, input params have names, but output params do not?

Reply:
I'm not sure if I understand the question. There's always just one return param, that can be None or a tuple… It does't have a name. But it is possible to define an alias for a type.

Reply:
wel, i mean here we can see that the first input param of the function is the image. But looking at what it returns we have no such names, we really have no clue what this could be. I figured it would be kinda nice if input and output params would be a bit symmetric here: that you could name both.